vulcanai2/models/dnn.py

```diff
@@ -106,16 +106,11 @@ def _forward(self, x, **kwargs):
         :return: output torch.Tensor
         """
 
-        if self._input_network and self._input_network.__class__.__name__ == "ConvNet":
-            x = x.view(-1, self._input_network.conv_flat_dim)
-
+        if self._input_networks and self._input_networks[0].__class__.__name__ == "ConvNet":
```

Comment:
you should check for more than just the first input network. need to go through all of them